sdks/java/io/elasticsearch-tests/elasticsearch-tests-common/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOTestUtils.java

```diff
@@ -80,7 +81,8 @@ class ElasticsearchIOTestUtils {
   /** Enumeration that specifies whether to insert malformed documents. */
   public enum InjectionMode {
     INJECT_SOME_INVALID_DOCS,
-    DO_NOT_INJECT_INVALID_DOCS
+    DO_NOT_INJECT_INVALID_DOCS,
+    INJECT_ONE_ID_TOO_LONG_DOC_AT_THE_END
```

Comment:
nit: I think this should be renamed to `INJECT_ONE_ID_TOO_LONG_DOC` because `AT_THE_END` is not guaranteed once the data is in a PCollection via `Create.of`